specification/security/resource-manager/Microsoft.Security/preview/2023-08-01-preview/pricings.json

```diff
@@ -0,0 +1,463 @@
+        },
+        "enforce": {
+          "type": "boolean",
+          "description": "Whether to allow descendants of this scope to not inherit the pricing set on this scope or not. Available only for subscription level pricing."
+        },
+        "inherited": {
+          "type": "boolean",
+          "description": "True if the pricing on this scope is inherited from a parent. The parent scope id will be in the \"source\" field.  Available only for resource level pricing."
+        },
+        "source": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The id of the scope inherited from. \"Null\" if not inherited. Available only for resource level pricing."
+        },
+        "extensions": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Extension"
+          },
+          "description": "Optional. List of extensions offered under a plan.",
+          "x-ms-identifiers": [],
+          "maxItems": 100
+        },
+        "deprecated": {
+          "type": "boolean",
+          "readOnly": true,
+          "description": "Optional. True if the plan is deprecated. If there are replacing plans they will appear in `replacedBy` property"
+        },
+        "replacedBy": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "readOnly": true,
+          "description": "Optional. List of plans that replace this plan. This property exists only if this plan is deprecated.",
+          "maxItems": 100
+        }
+      },
+      "required": [
+        "pricingTier"
+      ]
+    },
+    "Extension": {
+      "type": "object",
+      "description": "A plan's extension properties",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The extension name. Supported values are: <br><br>**AgentlessDiscoveryForKubernetes** - API-based discovery of information about Kubernetes cluster architecture, workload objects, and setup. Required for Kubernetes inventory, identity and network exposure detection, attack path analysis and risk hunting as part of the cloud security explorer.\r\nAvailable for CloudPosture plan.<br><br>**OnUploadMalwareScanning** - Limits the GB to be scanned per month for each storage account within the subscription. Once this limit reached on a given storage account, Blobs won't be scanned during current calendar month.\r\nAvailable for StorageAccounts plan.<br><br>**SensitiveDataDiscovery** - Sensitive data discovery identifies Blob storage container with sensitive data such as credentials, credit cards, and more, to help prioritize and investigate security events.\r\nAvailable for StorageAccounts and CloudPosture plans.<br><br>**ContainerRegistriesVulnerabilityAssessments** - Provides vulnerability management for images stored in your container registries.\r\nAvailable for CloudPosture and Containers plans."
```

Comment:
description looks like it contains formatting errors and markdown snippets. It is recommended to write succinct descriptions.